### ISSUE TITLE
Is it possible to stop to expand pointer field when it point to NULL (0x0)?

### ISSUE THREAD

**Issue by K9A2** [microsoft/vscode-cpptools#1705]
# Is it possible to stop to expand pointer field when it point to NULL (0x0)?

When we use integrated debugger, such as inspecting a link list in written in C with list node defined as below:

```C
struct ListNode {
    int val;
    struct ListNode *next;
}
```

We can expand its pointer field (next) as much as we want. But it could leed to possible confusion to noobs. Therefore, it is possible to stop user to at the end of list (i.e., remove the right triangle at the next field of node e090 to make it un able to expand it when it points to 0x0)?
![_20180320203757](https://user-images.githubusercontent.com/9523544/37655166-7a010486-2c7f-11e8-88f3-bd06aeb0770a.png)

**Comment by WardenGnaw** [comment]
Closing due to moving issue to MIEngine. https://github.com/Microsoft/MIEngine/issues/698

**Comment by sean-mcmanus** [comment]
@K9A2 A fix is available in the C/C++ extension 1.32.0 pre-release: https://github.com/microsoft/vscode-cpptools/releases/tag/v1.32.0 .
